SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorCloudLauncherSettings.h

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
What category does this settings appear under in the advanced settings? or spatial settings?

Reply:
With the latest change I added the settings to be part of the `Project Settings` 💯.